pytest_logger/plugin.py

```diff
@@ -195,6 +211,15 @@ def set_log_option_default(self, value):
         """ Sets default value of `log` option."""
         self._log_option_default = value
 
+    def divide_logs_by_result(self, div_by_res_logdir='div_by_res', div_by_res_subdirs=['failed']):
+        """Makes a directory inside main logdir where logs are further divided by test result
+
+        :param div_by_res_logdir: name for the subdirectory in main log directory
+        :param div_by_res_subdirs: list of test results to be handled (failed/passed)
```

Comment:
Outcomes - you can mention `skipped` as well.

Reply:
yeah I was thinking about it. but will `skipped` test ever produce any logs?

Reply:
Sure, it can - skipping can happen at the level of fixture or test itself. In both cases some logs may occur in fixtures or part of testcase:
https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions

Reply:
ok. `skipped` is also added